tslint.json

```diff
@@ -3,7 +3,10 @@
     "dist/src"
```

Comment:
Looks like your auto-formatter got to `tslint.json` 😉 please revert these changes.